src/components/Uploads/UploadsList.js

```diff
@@ -1,25 +1,31 @@
 import React from 'react';
-import styled from 'styled-components';
 import PropTypes from 'prop-types';
 import { UploadsListEmpty } from './UploadsListEmpty';
 import { UploadsListItem } from './UploadsListItem';
+import { List } from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
 
-const StyledContainer = styled.div`
-  display: flex;
-  flex-direction: column;
-`;
+const useStyles = makeStyles(() => ({
+  list: {
+    width: '100%',
+    height: '100%',
```

Comment:
why `100%` height?

Reply:
now it looks like this 
![image](https://user-images.githubusercontent.com/60269784/97790264-d8897780-1bcf-11eb-9e57-f056adf5c796.png)

this it's looks like this, if you remove height 100%
![image](https://user-images.githubusercontent.com/60269784/97790315-4afa5780-1bd0-11eb-806b-ca6b9bacefac.png)

Reply:
What if we have more upload items so that we need scroll? Will `height: 100%` allow scroll?
Maybe we need `min-height: 100%` ?

Also, why do we have to cover light gray background with dark gray background?
I mean why does the background of Uploads page is different than the default page background?
In the [design](https://www.figma.com/file/FxmwqO6LRg7tOVXrZqMMG0/Neural-Notes-Design-v.0.3?node-id=212%3A836) almost all pages have the same background as Uploads: dark gray. I don't see any page with the light gray background. Maybe we can just set the dark gray as default background?
Why do we need this light gray background beneath the dark gray background?

Reply:
> What if we have more upload items so that we need scroll? Will height: 100% allow scroll?
Maybe we need min-height: 100% ?

Agree, fixed it.

Replaced light gray to dark gray.